src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(x) {
+    count++;
+
+    return function(y) {
```

Comment:
I think the arrow function will look better here
```suggestion
  return (x) => {
    count++;

    return (y) => {
```